docs/uns-use-the-network/cli.md

```diff
@@ -635,3 +635,44 @@ $ uns did-resolve --network devnet @unik:individual/bob?postal_address -f raw
 
 42 quai Malakoff, 44000 Nantes
 ```
+
+### `send`
+
+#### Introduction
+Send owned UNS protocol tokens to another wallet.
+
+#### Arguments
+- `AMOUNT` (required):  The quantity of tokens to send to the recipient.
```

Comment:
we must specify that it's in UNS (not in a smaller unit of value)

Reply:
@laurentlourenco